static/js/latest-news.js

```diff
@@ -0,0 +1,63 @@
+(function() {
```

Comment:
This is copied from cn.ubuntu.com, and then heavily modified. For one thing, I made it IE11 compatible by removing `s and `=>` functions, and I wrapped it in an anonymous function, and made it so we could pass through the gtmEventLabel.